plugins/node/opentelemetry-instrumentation-cassandra/src/types.ts

```diff
@@ -29,4 +35,6 @@ export interface CassandraDriverInstrumentationConfig
    * @default 65536
    */
   maxQueryLength?: number;
+  /** Function for adding custom attributes before response is handled */
+  responseHook?: CassandraDriverResponseCustomAttributeFunction;
```

Comment:
This new instrumentation option should also be documented in the [README](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-cassandra#instrumentation-options) under "Instrumentation Options"